product_docs/docs/biganimal/release/reference/api.mdx

```diff
@@ -215,7 +215,7 @@ To call the BigAnimal API, your application must pass the retrieved access token
 
 ```shell
 curl --request GET \
-  --url "$AUDIENCE/v2/pg-types" \
+  --url "$AUDIENCE/v3/pg-types" \
```

Comment:
`$AUDIENCE/v3/pg-types` does not exist on v3, suggesting to change it to `$AUDIENCE/v3/users`